README.md

```diff
@@ -77,6 +77,8 @@ Run truffle, passing in the network e.g.
 
 
 ### Building and running the python tests
+We suggest running the following commands with an active python virtual environment ex. `virtualenv`.
```

Comment:
I think the standard way in python3 has moved to use `venv` instead

https://docs.python.org/3/library/venv.html#venv-def

Reply:
Thanks, changed it :)